guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -21,6 +21,13 @@ ifdef::client-content-dnf[]
 ----
 # {client-package-install-el8} katello-pull-transport-migrate
 ----
+ifndef::satellite[]
++
+Ensure that your host installs `yggdrasil` from {project-client-name}.
+If you have EPEL enabled on your host, you have to lock the version by using `echo "yggdrasil-_My_Package_Version_" >> /etc/dnf/plugins/versionlock.list` or filter the package by using a content view filter.
+You can find the `yggdrasil` package version on {project-client-name}.
+For more information, see {ContentManagementDocURL}Creating_a_Content_Filter_for_Yum_Content_content-management[Creating a content filter for Yum content] in _{ContentManagementDocTitle}_.
```

Comment:
This seems like too much information to be included as an afterthought after the `katello-pull-transport-migrate` step. Do you see a way to break it down into multiple steps perhaps?

Reply:
I have split this into two equally possible sub-steps. Is this what you had in mind?

Reply:
Yes! Seeing the information broken down into smaller chunks like this makes it a lot less intimidating at first glance. (Three-level nesting is not great but IMO that's the lesser evil.)